<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTiempoJuego">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tiempo:0 s</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemporizadorNivel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>7</width>
      <height>3</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Restante:0 s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVolcan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Nivel 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMamut">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSnow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMunicion">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Showcard Gothic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lblMunicion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIconoFlecha">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lblIconoFlecha</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
